frontend/components/form/formComponent.js

```diff
@@ -0,0 +1,142 @@
+import TextField from "@mui/material/TextField";
+import * as React from "react";
+import {Checkbox, FormControl, FormControlLabel, InputAdornment, InputLabel, MenuItem, Select} from "@mui/material";
+import Autocomplete from "@mui/material/Autocomplete";
+import AdapterDateFns from "@mui/lab/AdapterDateFns";
+import {DatePicker, LocalizationProvider} from "@mui/lab";
+import {useEffect} from "react";
+
+const InputTextField = (props) => {
+    // {...(error && {error:true,helperText:error})}
```

Comment:
clear this comment

Reply:
removed

Reply:
Please push the new commits here, so that we can see the update